website/content/docs/nia/api/api-overview.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync API
+sidebar_title: Overview
+description: >-
+  How to use the Consul-Terraform-Sync API
+---
+
+# Consul-Terraform-Sync API
+
+When running in [daemon mode](/docs/nia/cli#daemon-mode), Consul-Terraform-Sync serves an HTTP API interface. Details of the available API endpoints is available in the navigation to the left.
+
+### Port
+
+The API is served at the default port `8558` or a different port if set with [`port` configuration](/docs/nia/configuration#port)
+
+### Version Prefix
+
+All API routes are prefixed with `/v1/`. This documentation is for v1 of the API, which is the only version currently.
+
+Example: `localhost:8558/v1/status`
+
+### Error
+
+Successful API requests will receive a 2XX success status code. For other unsuccessful status codes, when possible, more details will be provided in a response body. The response will be a JSON map with an "error" key.
+
+Example: Status 400 Bad Request
+```json
+{
+  "error": "example error message: unsupported status parameter value"
```

Comment:
```suggestion
  "error": {
    "message": "example error message: unsupported status parameter value"
  }
```

Reply:
If you don't mind adding a change for me too! I forgot until I saw the diffs

Reply:
Thanks for thinking of this! Of course I don't mind. I actually have another PR prepared of miscellaneous API changes and have a commit for this. If it's ok with you, I'll leave this change for that PR so I can avoid waiting and rechecking vercel?

Reply:
That works too thank you